### ISSUE TITLE
The images on certain pages have no alt-text. Alt-text provides a description of the image to the screen reader user.

### ISSUE THREAD

**Issue by Paul-Clue** [LLK/scratch-www#7178]
### Where To Find Bug
Homepage, Explore, Ideas, MyStuff, Account Settings, Profile

### Expected Behaviour
If alt-text is present, the screen reader will read it for the user.

### Actual Behaviour
No alt-text is present on the image tags, therefore, the screen reader will read the names of the image files.

### Steps To Reproduce

- Open https://scratch.mit.edu/ in any browser on your computer
- Turn-on your screen reader of choice
- Using interactive mode, tab through the page until focus is on the first image on the page
- Notice that the screen reader announces the name of the image file instead of giving a description of the image.

### System Details
macOs Ventura, Chrome Browser, VoiceOver (AT)
